src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write your code here
+const logoClickedMessage = 'Logo clicked';
+const timeGone = 'Time gone';
+
+const resolver = (resolve, reject) => {
+  let clickedLogo = false;
+
+  document.getElementsByClassName('logo')[0].addEventListener('click', () => {
+    clickedLogo = true;
+    resolve(logoClickedMessage);
+  });
+
+  setTimeout(() => {
+    if (!clickedLogo) {
+      reject(timeGone);
+    }
+  }, 3000);
+};
+
+const promise1 = new Promise(resolver);
+const promise2 = new Promise(resolver);
+
+function addHandler(promise) {
```

Comment:
Consider renaming the 'addHandler' function to something more descriptive, like 'addPromiseHandlers'.